driver/controller.go

```diff
@@ -160,14 +159,23 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		klog.Infof("creating volume from snapshot %q", snapshotTarget.ID.String())
 	}
 
-	var sizeInBytes int64 = MinimalVolumeSizeBytes
+	var sizeInGiB int64 = DefaultVolumeSizeGiB
 	if req.GetCapacityRange() != nil {
-		sizeInBytes = req.GetCapacityRange().RequiredBytes
+		requiredBytes := req.GetCapacityRange().RequiredBytes
```

Comment:
Where are the `RequiredBytes` coming from?

Reply:
It's what the user specifies in their PVC manifest:
```yaml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: my-pvc
  namespace: awesome
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 200Gi
  storageClassName: exoscale-bs
```
The value 200Gi is converted to bytes by k8s.

Unrelated remark: It is possible for the user to specify a sub-GiB value here, for this case I introduced that the CSI will produce an error event. If the size is not within the block storage permitted range, we rely on the API to error.

Reply:
> Unrelated remark

That was exactly the reason for my question. I was wondering how a user could introduce a non GiB value.
Thanks!